### ISSUE TITLE
Not able to pull stock prices for NSE

### ISSUE THREAD

**Issue by vipuljain11** [lukaszbanasiak/yahoo-finance#119]
from yahoo_finance import Share
yahoo = Share('TCS.NS')
print (yahoo.get_open())
print (yahoo.get_price())

trying to pull the price for TCS.NS, however getting None as a output.

I am new to the python

**Comment by grafa** [comment]
You have to use only the ticker as parameter to `Share` class. So I guess in your case it's `TCS`.

**Comment by vipuljain11** [comment]
Thank you for your reply. However, I am looking for the price of the Tata consultancy services (https://in.finance.yahoo.com/quote/TCS.NS/?p=TCS.NS)  which is traded on NSE stock exchange. While 'TCS'  is pulling the data for  The Container Store Group, Inc.(https://in.finance.yahoo.com/quote/TCS?p=TCS)

**Comment by grafa** [comment]
when I try the quesry directly in YQL console it gives me the results:
https://yhoo.it/2nEbMDJ

Try to debug down the problem in the library itself.

**Comment by psytronoxious** [comment]
@vipuljain11 Yes. You are right. I tried 'SBIN.NS', it didn't work. I also tried 'CENTRICA.IL' and 'CENTRICA.L' from London Stock Exchange, even it didn't work. Till then I have to get data from 
https://chartapi.finance.yahoo.com/instrument/1.0/SBIN.NS/chartdata;type=quote;range=1d/csv.
Any help is appreciated.
